tools/UpdatePaths/Scripts/23379_food_containers.txt

```diff
@@ -0,0 +1,2 @@
+/obj/item/reagent_containers/food/drinks : /obj/item/reagent_containers/drinks/@SUBTYPES{OLD}
+/obj/item/reagent_containers/food : /obj/item/food/@SUBTYPES{OLD}
```

Comment:
These should have `/@SUBTYPES` on the before replacement too I think i.e.
/obj/item/reagent_containers/food/@SUBTYPES : /obj/item/food/@SUBTYPES{OLD}

Reply:
Yeah you're right, added!